### ISSUE TITLE
Help with Optimization of Settings?

### ISSUE THREAD

**Issue by Buddhaman42** [AHAAAAAAA/PokemonGo-Map#3246]
Hey everyone....first just want to say thank you to everyone here. Without you all, I would never have gotten as far as I have. That being said......I need some help optimizing my overall setup. I am currently using the development code. I have 15 accounts setup so far. I have tried running them all from one location with a large step size and each account on its own script separately in a gridlike pattern (shown below). I can cover a wider area with the individual accounts and smaller step size, so I think I want to go from there. The picture below is using -st 7 -sd 8 -t 1. I have tried increasing/decreasing st and sd slightly with little change in results. 

The issue I am having as you can see is that the grids do not seem to complete and have many missing "holes". They also appear to vary in size despite having the same step size.  In the command box I frequently get the following warning: "[warning] database is locked... Retrying". After a few rounds of that it searches normally and then repeats that cycle.

Basically, I would like it if anyone could help me figure out ways to make my setup more efficient. Do I just need more accounts to run multiple accounts per location? Any other tips? Thank you so much in advance!

![image](https://cloud.githubusercontent.com/assets/20763688/17351046/73834b74-58f9-11e6-8d2b-fe0083248354.png)

**Comment by InflamedSebi** [comment]
You are using the standard SQLite Database, right? SQLite is actually a single user database. It will work with more users (each running script =1 user) as long as they don't access it the exact same time.But multiple instances could collide and one of the instances would not be able to write the data and discard it after a time out.
You can avoid this by using a mysql Database or any equivalent multiuser DB.

´-t´ doesn't exist in the development branch anymore and should show an error if used. are you sure you are using dev branch?
´-sd 5´ (lower = faster) is the minimum you can use to get all pokemon to show up.

**Comment by Buddhaman42** [comment]
Thanks. I was using SQlite and now have mysql set up and working.......

It seems to be working OK, but I would still like it to be better. From the config file all of my accounts are running from the same location, but I would prefer it to be from multiple locations like I have above. Is there a way using MySQL to run from several locations? Thanks.

**Comment by InflamedSebi** [comment]
multiple locations don't work with the config sadly ... already requested that feature.
but you can remove the from the config completely and add them to the commandline:

`python runserver.py -l somelocation -a ptc -u user1 -p pass1 -a ptc -u user2 -p pass2 ...and_so_on... `

**Comment by Buddhaman42** [comment]
and that will automatically run through MySQL now?

**Comment by InflamedSebi** [comment]
ya all instances of runserver.py will use the same config, so all use the same database.
